README.md

```diff
@@ -113,7 +113,7 @@ of SimpliSafe™ systems. Two types of objects can be returned:
 Despite the differences, `simplipy` provides a common interface to
 these objects, meaning many of the same properties and methods are available to both.
 
-### Base Properties and Methods
+### V3 Properties
```

Comment:
Why describe V3 _before_ base?

Reply:
Good point. Mainly because people may want code samples that show the full capability of their system.

Reply:
I think this actually makes things more confusing. It _also_ doesn't show "full" capability, since those base methods and properties aren't added to this section (and are still in their own section).

Long story short, I'm not convinced this actually solves a problem.

Reply:
Like I said earlier, I think Read The Docs would be the best choice.

Reply:
Maybe you can branch out to a SPECIFIC-DOCS.md file?

Reply:
I appreciate the thought, but no other user has suggested such a large change to the docs. `simplisafe-python` isn't large enough to warrant RTD at this time.